docs/platformdev_rp2040.md

```diff
@@ -4,7 +4,7 @@ The following table shows the current driver status for peripherals on RP2040 MC
 
 |                              System                              |                    Support                     |
 | ---------------------------------------------------------------- | ---------------------------------------------- |
-| [ADC driver](adc_driver.md)                                      | Support planned (no ETA)                       |
+| [ADC driver](adc_driver.md)                                      | :heavy_check_mark:                             |
```

Comment:
Making this change in the `master` branch is not correct, because the ADC support for RP2040 is not present there yet (it should appear after the next round of breaking changes, which is scheduled for February 26, 2023). Adding a note about that might make sense though (but apparently in other similar cases this was not done).